src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl_constraints.erl

```diff
@@ -60,19 +64,28 @@ check_admins_group_removal_from_grant_ace(OrgId, AuthzId, Type, AclPerm, NewAce)
     <<"grant">> ->
       NewGroups = extract_acl_groups(AclPerm, NewAce),
       CurrentAce = oc_chef_authz_acl:fetch(Type, OrgId, AuthzId),
-      CurrentGroups = extract_acl_groups(AclPerm, CurrentAce),
-      case check_admins_group_removal(CurrentGroups, NewGroups) of
-        not_removed ->
-          false;
-        removed ->
-          {true, attempted_admin_group_removal_grant_ace}
+      %% CurrentAce always = {error, _} ?
+      % TODO: revert the changes to this case statement and see what happens
+      case CurrentAce of
+          {error, _} ->
+              error;
+          % Dialyzer says this will never match
```

Comment:
I think we've fixed this so we can remove the comments here.

Reply:
Done.